renderdoc/driver/d3d11/d3d11_counters.cpp

```diff
@@ -132,11 +237,15 @@ void D3D11DebugManager::FillTimers(CounterContext &ctx, const DrawcallTreeNode &
 
     m_pImmediateContext->Flush();
 
-    if(timer->before && timer->after)
+    if(timer->before && timer->after && timer->stats)
```

Comment:
Ditto checking occlusion here for NULL, but it might be better to check each one individually (in case for some reason the stats failed to create but the timer succeeded, at least we get that data).